src/core/radio/radio.hpp

```diff
@@ -62,6 +63,13 @@ static constexpr uint64_t kMinCslPeriod  = OPENTHREAD_CONFIG_MAC_CSL_MIN_PERIOD
 static constexpr uint64_t kMaxCslTimeout = OPENTHREAD_CONFIG_MAC_CSL_MAX_TIMEOUT;
 #endif
 
+#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
+/**
+ * Minimum WED listen duration supported in us.
+ */
+static constexpr uint32_t kMinWedListenDuration = 100;
```

Comment:
This value is picked to make sure too low values which could cause issues when scheduling receive slots are not selected. But I would leave the possibility open to pick values shorter than the wake-up frame duration for testing purposes, for instance. Notice that some radios are able to fully receive an incoming frame when only the preamble has been received during an scheduled reception.

Since the default configuration is `kDefaultWedListenDuration` anyway, I would keep the minimum configurable value to something small like 100 us.